src/content/docs/configuration/target-config.mdx

```diff
@@ -28,7 +28,7 @@ Daytona provides an option to set a Target Configuration to use when managing Wo
 1. Run the following command to set a Target Configuration:
 
 ```shell
-daytona target set
+daytona target-config set
```

Comment:
We removed support for editing target configs. Once added, it can only be removed.

So the only 2 commands for TC are: `add` and `remove` (+ `list`).

Also, there is no setting a default TC. A Target can be set as default and has the `--target` creation flag.